### ISSUE TITLE
Resource pack and support

### ISSUE THREAD

**Issue by PriestOfTerror** [PluginBugs/Issues-ItemsAdder#2395]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

PriestOfTerror#9992

### What happened?

tell me where i can post problems. Your site is confusing.

I bought a plugin from you but I don't know where to get the resource pack, can you please advise where I can find it?

### Steps to reproduce the issue

help

### ItemsAdder Version

last

### ProtocolLib Version

last

### LoneLibs Version

last

**Comment by MelonCraftDE** [comment]
https://itemsadder.devs.beer/first-install

**Comment by PriestOfTerror** [comment]
Okay, thaks I have it allready, but it doesnt work, after connecting with dropbox, can u please tell me, where somebody can help me? 
Because I cant find discord server what is in description and need help from somebody

Best regrads 
David Hačko

**Comment by LoneDev6** [comment]
- Open [DropBox](https://dropbox.com/), register/login
- Use the command `/iazip`
- Open folder: `plugins/ItemsAdder/output/`
- Drag and drop on DropBox the file `generated.zip`
- Press Share
![image](https://user-images.githubusercontent.com/27242001/223394446-819b4147-93b0-4681-b550-4b0a5ee74d52.png)

- Press Create

![image](https://user-images.githubusercontent.com/27242001/223394483-97215355-1d6e-4d23-91ff-02914f3580a0.png)

- Press Copy link
- For example if your link is `https://www.dropbox.com/blablabla?dl=0`
- Open config.yml of ItemsAdder
- Set it like this (I used the example URL, please use your own)

```yml
resource-pack:
  apply-on-join: true
  kick-player-on-decline: false
  delay-ticks: 1
  self-host:
    enabled: false
  external-host:
    enabled: true
    url: 'https://www.dropbox.com/blablabla?dl=0'
```

- run `/iareload` to reload the config
- run `/iatexture` in-game to apply the resourcepack to your game

**Comment by PriestOfTerror** [comment]
Yes, I did it exactly like this, but still doesnt work

**Comment by LoneDev6** [comment]
Provide a detailed explanation with screenshot of what "doesn't work" means

**Comment by PriestOfTerror** [comment]
Hello, after instaling I have problem with figures, I can see skin, but no figure

**Comment by PriestOfTerror** [comment]
![image](https://user-imag
![image](https://user-images.githubusercontent.com/50196238/224166063-5a1da33e-4bb1-4141-87f7-1dcca5cb1295.jpg)
es.githubusercontent.com/50196238/224166034-d5912a6d-a515-4453-a1bf-ca4e177c12e6.jpg)

**Comment by LoneDev6** [comment]
It's planned to be fixed on 3.4.1b.
It's a MacOS bug, I hope to manage to find a way to fix that.
https://github.com/PluginBugs/Issues-ItemsAdder/issues/2182
